### PR TITLE
Ed branch

### PR DESCRIPTION
**Pull request by edCitalan**
Load method & unit testing

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,11 +1,29 @@
-API_KEY="api_key"
```
Comment (jhill00) [inline .env:1]: remove the personal info like api key and your postgres password for example.

File: etl_project/pipelines/w.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+if __name__ == "__main__":
+    loads = load_dotenv()
+    API_KEY = os.environ.get("API_KEY")
+    DB_USERNAME = os.environ.get("DB_USERNAME")
+    DB_PASSWORD = os.environ.get("DB_PASSWORD")
+    SERVER_NAME = os.environ.get("SERVER_NAME")
+    DATABASE_NAME = os.environ.get("DATABASE_NAME")
+    PORT = os.environ.get("PORT")
+
+    # API_KEY = os.getenv("API_KEY")
```
Comment (jhill00) [inline etl_project/pipelines/w.py:17]: not the biggest deal, but why the commented out environment variables right below the environment variables?

**Review by jhill00 — CHANGES_REQUESTED** [review]
Great work Edward, that was a lot you put together. Just change the environment variables to values that are not your personal info, e.g. api key, pg password, etc and I'll approve it

**Review by jhill00 — APPROVED** [review]

**Review by notyalc — APPROVED** [review]
Looks good thanks!

**Pull request merged**